src/ClearAll.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
- [ ] Please don't disable linters. They are helpful in ensuring that your code adheres to best practices and coding conventions👍